rasa/core/policies/ensemble.py

```diff
@@ -737,6 +707,64 @@ def _check_policy_for_forms_available(
         )
 
 
+def _training_events_from_trackers(
```

Comment:
the name doesn't seem to correlate with what this is doing. Can we please find a better name? `_side_effects_per_action` or something?